docs/conf.py

```diff
@@ -33,3 +33,5 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'OpenHatchdoc'
 
+# Stop converting quotes and dashes to typographically correct entities. Useful for placing code into notes, ie.
+html_use_smartypants = False
```

Comment:
I should probably mention that this will kill all of our fancy directional quotes. I think correctness here is more important than things looking slightly prettier, but if someone wants to investigate a different fix please feel free.